.eslintrc.js

```diff
@@ -19,7 +19,12 @@ module.exports = {
     sourceType: "module",
   },
   rules: {
-    "@typescript-eslint/no-non-null-assertion": 0,
```

Comment:
I had a feeling we weren't triggering this rule and we weren't time to remove it so we don't start skipping it unintentionally.